bsb_test/parallel.py

```diff
@@ -89,16 +89,18 @@ def timed_f(*args, **kwargs):
                     del _exc_threads[hash(thread)]
                     raise e
             except Exception as e:
-                import traceback, sys
-
-                errlines = traceback.format_exception(type(e), e, e.__traceback__)
-                print(
-                    *errlines,
-                    file=sys.stderr,
-                    flush=True,
-                )
                 if MPI.get_size() > 1:
```

Comment:
This test seems to be not sufficient to differentiate the cases in parallel and in serial.